modules/diskq/qdisk.c

```diff
@@ -509,6 +508,8 @@ _load_queue(QDisk *self, GQueue *q, gint64 q_ofs, guint32 q_len, guint32 q_count
             {
               msg_error("Error reading message from disk-queue file (maybe corrupted file) some messages will be lost",
                         evt_tag_str("filename", self->filename),
+                        evt_tag_uint("num_of_messages", q_count),
+                        evt_tag_long("invalid_index", i),
```

Comment:
`evt_tag_long` might not be the best for `gsize`, which is `unsigned long`.

Reply:
The name of the function is misleading: it is actually formatted as `"%lld"`

```
EVTTAG *
evt_tag_long(const char *tag, long long value)
{
  char buf[32]; /* a 64 bit int fits into 20 characters */

  snprintf(buf, sizeof(buf), "%lld", value);
  return evt_tag_str(tag, buf);
}
```

Reply:
Shouldn't we print `unsigned long` with `%llu` or `%lu`?

Reply:
I changed my mind, and will use `guint32` for `i`, which fixes the original problem.

Also, I was confused with `evt_tag_long` too, that's why I introduced `evt_tag_uint`. On the other hand, every value here can be represented with `evt_tag_long`, so I dropped that patch too.